### PR TITLE
WIP: Add tutorial for multilabel training

### PR DESCRIPTION
**Pull request by PariaValizadeh**
This PR adds a tutorial notebook to guide practictioners how to train a ResNet model for multilabel classification on the HSN dataset.

### CODE REVIEW

File: notebooks/tutorials/additional_tutorials/Multilabel_ResNet_on_HSN.ipynb [hunk 1]
```diff
@@ -0,0 +1,1155 @@
...
+    "        num_frames=(hsn_test[sample]['end_time'] - hsn_test[sample]['start_time']) * sr\n",
+    "        )\n",
+    "sample_audio = sample_audio[0]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Step 6: Visualization"
```
Comment (raphaelschwinger) [inline notebooks/tutorials/additional_tutorials/Multilabel_ResNet_on_HSN.ipynb:436]: @PariaValizadeh Could you move the visualization before the model training? So that the reader sees how the data gets preprocessed by the birdset pipeline?

**Review by raphaelschwinger — CHANGES_REQUESTED** [review]
Hey @PariaValizadeh,
Thanks for your work. To get the most out it for our readers, I would suggest the following changes:
- Load the data with our `BirdSetDataModule` without any preprocessing so that it outputs batched waveforms
- Pick the first sample of the batch from the train and test set (this will show the difference of focal / soundscapes)
- Make the sounds audible
- Now, add preprocessing to train a EfficientNet (EfficientNet because we can upload a pretrained model for that, and we included the EfficientNet in BirdSet), make sure to use the same preprocessing config as done in `configs/experiment/birdset_neurips24/HSN/DT/efficientnet.yaml`
- Visualize the first sample (now a spectrogram)
- Train the model (with same configs) and add the option to download the pretrained model
- Run test with `trainer.test()`
- Run model on previously selected test sample
- Print the predicted classes and the target class.
